### PR TITLE
fix(frontend): add missing env to build globals

### PR DESCRIPTION
**Pull request by guerrap**

### CODE REVIEW

File: packages/frontend/package.json [hunk 1]
```diff
@@ -21,8 +21,10 @@
     "scripts": {
         "lint": "eslint --ext .js,.ts,.tsx ./ && prettier --check .",
         "format": "eslint --fix --ext .tsx,.ts,.js ./ && prettier --write .",
-        "build": "node ./scripts/build.js",
-        "build:prod": "NODE_ENV=production node ./scripts/build.js",
+        "build": "yarn build:dev",
+        "build:production": "NODE_ENV=production ENVIRONMENT=prod node ./scripts/build.js",
+        "build:staging": "ENVIRONMENT=staging node ./scripts/build.js",
```
Comment (luzzif) [inline packages/frontend/package.json]: Maybe we should build production code also for staging as it's meant to be an as-close-as-possible environment to production.

File: packages/frontend/src/creation-form/index.tsx [hunk 2]
```diff
@@ -67,12 +67,11 @@ export const Component = ({
     const { t } = useTranslation();
     const devMode = useDevMode();
     const templatePreviewMode = useTemplatePreviewMode();
-    // FIXME: use the appropriate answerer service based on the current environment
     const answererUrl = useMemo(() => {
         return devMode
             ? "http://127.0.0.1:9080"
             : templatePreviewMode
-              ? `https://defillama-answerer.api.dev.carrot.community`
+              ? `https://defillama-answerer.${ENVIRONMENT}.dev.carrot.community`
```
Comment (luzzif) [inline packages/frontend/src/creation-form/index.tsx]: Pretty sure this should replace `dev`, not `api`

**Review by luzzif — CHANGES_REQUESTED** [review]

File: packages/frontend/src/creation-form/index.tsx (hunk 2 above)
Comment (guerrap) [reply packages/frontend/src/creation-form/index.tsx]: Oh yikes, yep.

File: packages/frontend/package.json (hunk 1 above)
Comment (guerrap) [reply packages/frontend/package.json]: Fixed in 9e02924b0b2b9b66b7c58af79aee469fbaceca85.

File: packages/frontend/src/creation-form/index.tsx (hunk 2 above)
Comment (guerrap) [reply packages/frontend/src/creation-form/index.tsx]: Fixed in 9e02924b0b2b9b66b7c58af79aee469fbaceca85.

**Pull request merged**